flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
Indeed, I am limited by the wheels provided by upstream projects and flatpak basic builder handling of python packages.

Reply:
I beg to disagree

![image](https://user-images.githubusercontent.com/114441/181996637-6b25e8a8-1ece-4827-ac92-ac2cd72d261c.png)

![image](https://user-images.githubusercontent.com/114441/181996659-b1ef3f10-0abe-416f-976c-11b6c0a2a8ac.png)

Reply:
Is this is blocking to get it published on flathub, if so, do you mind linking where it is stated in the documentation?